velox/dwio/text/writer/TextWriter.cpp

```diff
@@ -139,73 +150,95 @@ void TextWriter::writeCellValue(
         serDeOptions_.nullString.data(), serDeOptions_.nullString.length());
     return;
   }
+
+  ++depth_;
```

Comment:
Let's not put depth in member variable but use a local variable in `write` function